db/migrate/20231114000534_create_recipe_foods.rb

```diff
@@ -0,0 +1,11 @@
+class CreateRecipeFoods < ActiveRecord::Migration[7.0]
+  def change
+    create_table :recipe_foods do |t|
+      t.references :food, null: false, foreign_key: true
+      t.references :recipe, null: false, foreign_key: true
+      t.integer :quantity
+
+      t.timestamps
+    end
```

Comment:
- [ ] Also I would suggest you follow the hint he gave to resolve the issue of the recipe in the `create_recipe_food` table as I also get an error when trying to migrate the file to create a copy of the database on my machine.

Error message 👇 

![image](https://github.com/Gardimy/ror_recipe_app/assets/78634841/8d06950c-2d2c-404c-88b3-01a8c456b0de)

Reply:
I Don't get any error when I  run the migration file.

Here is the screenshot:
![Capture](https://github.com/Gardimy/ror_recipe_app/assets/111473989/56137ab0-31d9-40aa-a8c0-853607d3287a)


NB: when I make the change you provided, I get a lot of issues.